plugins/inventory/digitalocean.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c), Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+name: digitalocean
+plugin_type: inventory
+author:
+  - Janos Gerzson (@grzs)
+short_description: DigitalOcean Inventory Plugin
+version_added: "1.1.0"
+description:
+  - DigitalOcean (DO) inventory plugin.
+  - Acquires droplet list from DO API.
+  - Uses configuration file that ends with C((do_hosts|digitalocean|digital_ocean).(yaml|yml)).
+extends_documentation_fragment:
+  - community.digitalocean.digital_ocean.documentation
+  - constructed
+  - inventory_cache
+options:
+  plugin:
+    description:
+      - The name of the DigitalOcean Inventory Plugin,
+      - this should always be 'community.digitalocean.digitalocean'.
```

Comment:
Other places should be changed though. Also, this is part of the previous sentence, not a new paragraph, so removing the `-`:
```suggestion
        this should always be C(community.digitalocean.digitalocean).
```